### DISCUSSION TITLE
[Icon request] Apps

### DISCUSSION THREAD

**Discussion by mpskovvang** [tailwindlabs/heroicons#478] [Ideas]
An apps icon would be really useful in a project I'm currently working on to show all the additional tools/features that don't fit in the navigation bar. 😎

Inspiration from Material Design icons:
![outline_apps_black_18dp](https://user-images.githubusercontent.com/7034075/92815266-5bf2cd80-f3c4-11ea-82f8-cc45d6dfc495.png)

Another use-case taken from Google Chrome:
![image](https://user-images.githubusercontent.com/7034075/92815421-82186d80-f3c4-11ea-9494-c0e1da334b28.png)

**Comment by ps73** [comment]
You can use the "view-grid" icon, but it is 2x2 not 3x3 layout

**Comment by mpskovvang** [comment]
Thanks for your suggestion @ps73. I did consider the same, but the `view-grid` icon is already used in a more common place (list/grid toggle). It might harm UX using the same icon for two different things.
